### DISCUSSION TITLE
Configuration section in README

### DISCUSSION THREAD

**Discussion by RaZ0rr-Two** [davatorium/rofi#1529] [Q&A] [answered]
Currently in [this section](https://github.com/davatorium/rofi#configuration) in README, the link provided is to [this file](https://github.com/davatorium/rofi/blob/next/CONFIG.md) in the repository.

Should this be changed back [to this](https://github.com/davatorium/rofi/wiki/Configuring-Rofi) page in the wiki? ( copy paste text from CONFIG.md to wiki page and delete CONFIG.md)

The [themes section](https://github.com/davatorium/rofi/wiki/themes) is also in the wiki so maybe I think configuration should be put there too.

What do you think?

**Comment by DaveDavenport** [comment, marked as answer]
nobody is maintaining the wiki.
